src/smart-components/role/add-role-new/review.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-form-api';
+import {
+    Stack,
+    StackItem,
+    Grid,
+    GridItem,
+    Text,
+    TextVariants
+} from '@patternfly/react-core';
+import './review.scss';
+
+const mockData = [
+    { cells: [ 'cost:aws.account:read', 'Project 1' ]},
+    { cells: [ 'cost:azure:read', 'Cluster 1, Cluster 2, Cluster 3' ]},
+    { cells: [ 'cost:aws.account:write', 'Project 1' ]},
+    { cells: [ 'cost:aws.account:execute', 'Project 1' ]},
+    { cells: [ 'cost:aws:something', 'Project 1' ]},
+    { cells: [ 'cost:aws:something2', 'Project 2' ]},
+    { cells: [ 'cost:aws:something3', 'Project 3' ]},
+    { cells: [ 'cost:aws:something4', 'Project 4' ]},
+    { cells: [ 'cost:aws:something5', 'Project 5' ]},
+    { cells: [ 'cost:aws:something6', 'Project 6' ]},
+    { cells: [ 'cost:aws:something7', 'Project 7' ]},
+    { cells: [ 'cost:aws:something8', 'Project 8' ]},
+    { cells: [ 'cost:aws:something9', 'Project 9' ]},
+    { cells: [ 'cost:aws:something10', 'Project 10' ]},
+    { cells: [ 'cost:aws:something11', 'Project 11' ]},
+    { cells: [ 'cost:aws:something12', 'Project 12' ]}
+];
+
+const table = (columns, rows) => (
+    <div className='sticky'>
+        <section className="title">
+            {columns.map(col => <span key={ col }>{col}</span>)}
+        </section>
+        <section className="data">
+            {rows.map(row => (
+                <div key = { `row-${row.cells}` } className='row'>
+                    {row.cells.map(cell => <span key={ cell }>{cell}</span>)}
+                </div>
+            ))}
+        </section>
+    </div>
+);
+
+const ReviewStep = () => {
+    const formOptions = useFormApi();
+    const {
+        'role-name': name,
+        'role-description': description,
+        'role-copy-name': copyName,
+        'role-copy-description': copyDescription,
+        'add-permissions-table': permissions
+    } = formOptions.getState().values;
+    const columns = [ 'Application', 'Resource type', 'Operation' ];
+    const rows = permissions.map(permission => ({
+        cells: permission.uuid.split(':')
+    }));
+
+    return <React.Fragment>
+        <Stack hasGutter>
+            <StackItem className="ins-c-rbac__summary">
+                <Grid>
+                    <GridItem span={ 2 }>
+                        <Text component={ TextVariants.h4 } style={ { 'font-weight': 'bold' } }>Name</Text>
```

Comment:
Can you add this styling to class files instead? Dirct styling is not that good.